bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_YTobb_HToWWTo2QLNu/NMSSM_XYH_YTobb_HToWWTo2QLNu_MX_mMX_MY_mMY_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+import model NMSSMHET_UFO --modelname
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = p
```

Comment:
The model is per se 4FS. It makes no sense to just add the b in the definition of the p. Please add a restriction card with massless b if you want to use 5FS.